maestro-inspector/src/main/java/org/maestro/inspector/base/InspectorManager.java

```diff
@@ -110,6 +113,19 @@ public void handle(SetRequest note) {
     public void handle(StopInspector note) {
```

Comment:
I don't like this solution. The problem is that **writeTestProperties()**  and **writeSystemProperties()** are part of the **MaestroWorkerManager** class. Inspector extends this class so it's no problem to use it there, but log directory cause the problem because it's not synced with Inspector instances (Artemis/Interconnect). Possible solution is to create writer for system info which will be available inside the specific inspector implementations. Let me know what do you think about it.